src/main/java/com/google/flourbot/execution/ChatResponse.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Helps control what card is displayed by bot after macro execution.
+ */
+
+package com.google.flourbot.execution;
+
+import com.google.flourbot.entity.action.ActionType;
+
+import java.util.Collection;
+import java.util.List;
+
+public class ChatResponse {
```

Comment:
This class and its methods could really use some javadoc because it is not clear what is the goal here... one thing to consider is that for the simplest responses we may want to simply return the text.

Reply:
Got it!